src/Cake.Common/Tools/NuGet/Sources/NuGetSourcesSettings.cs

```diff
@@ -44,5 +45,10 @@ public sealed class NuGetSourcesSettings : ToolSettings
         /// <c>true</c> if password is stored as unencrypted; otherwise, <c>false</c>.
         /// </value>
         public bool StorePasswordInClearText { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value the NuGet configuration file. If not specified, file %AppData%\NuGet\NuGet.config is used as configuration file.
```

Comment:
@devlead that wording here is a little off.  Perhaps `Gets or sets the location of the NuGet configuration file`

Reply:
agreed I'll fix.

Reply:
this should be addressed now.